Run-SystemTest.ps1

```diff
@@ -0,0 +1,37 @@
+#Requires -Version 5.0
```

Comment:
I have issue running it locally, is there anything special I need to do? 
![image](https://github.com/gsoft-inc/wl-openapi-msbuild/assets/8277266/454a4b3e-48f2-498e-8943-543fe87f6138)
 (if yes we could put the instructions in the Readme)

Reply:
Updated file to remove `.nuget` cache clear. This is what was causing this error